server/.env

```diff
@@ -0,0 +1,6 @@
+# These are environment variables for development in local machine.
```

Comment:
If that's the case should it be `.env.development` or `.env.development.local` instead?

Reply:
Ohh cos dotenv cannot read `.env.development` :/

Reply:
Okay actually dotenv-flow can read `.env.development`.